### ISSUE TITLE
Shell command output not properly bounded by grey rect

### ISSUE THREAD

**Issue by mikevel** [progit/progit2#133]
It seems that the output of certain shell commands overruns the grey bounding rectangle. Here's an example from the bottom of page 43 of the PDF:
![image](https://cloud.githubusercontent.com/assets/1465262/4859226/dd4f8196-60e7-11e4-957f-6906e72f218a.png)

**Comment by gartenfeld** [comment]
@schacon This remains an issue particularly for long shell commands, on Page 41 of PDF:
![overflow](https://cloud.githubusercontent.com/assets/1134883/13238718/dee10cde-d989-11e5-9ce8-b91af5ff3f21.png)

**Comment by HonkingGoose** [comment]
@ben @schacon This doesn't appear to be an issue anymore, maybe this issue can be closed?

---

I've skimmed trough [version 2.1.138](https://github.com/progit/progit2/releases/download/2.1.138/progit.pdf) of the .pdf. The command:
> $ git config --global core.editor "'C:/Program Files/Notepad++/notepad++.exe'
-multiInst -nosession"

is properly wrapped in a box (See page 22 in the new version of the pdf). This line is not broken up with a additional newline in the progit2/master source code.

I saw no other text that was not bounded by a box where it should be, on my admittedly quick skim trough the .pdf.
